bin/check_samplesheet.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
```

Comment:
This script got added but you are not using it though right? It is still the pipeline default.

Reply:
Yes, I do not use it. Ampliseq parses not just a sample sheet but also allows pointing to a folder or a fasta file with sequences to classify, all with --input. It might be possible to adjust this to the needs here, but `parse_samplesheet.nf ` below does this job already.

Reply:
Ok, thats postponed for now.